examples/esp32/components/hw_crypto/hw_crypto.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2022 Felix Gohla, Konrad Hanff, Tobias Kantusch,
+ *                    Quentin Kuth, Felix Roth. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+
+#include <sdkconfig.h>
+
+#ifdef CONFIG_USE_HW_CRYPTO
+#include "fido.h"
+
+#include <stdio.h>
+#include <mbedtls/aes.h>
+#include <mbedtls/gcm.h>
+#include <mbedtls/sha256.h>
+#include <mbedtls/sha512.h>
+
+int sha256(const uint8_t *data, size_t data_len, uint8_t *hash) {
```

Comment:
Those could be static, couldn't they?

Reply:
As they are only referenced in the init function below, yes I guess. Adjusted that for the nRF as well.